apps/api/src/domain/models/__tests__/utilities/getValidResourceAndContextPairForTest.ts

```diff
@@ -7,9 +7,6 @@ import { isNullOrUndefined } from '../../../utilities/validation/is-null-or-unde
 import { EdgeConnectionContextType } from '../../context/types/EdgeConnectionContextType';
 import { Resource } from '../../resource.entity';
 
-/**
- * TODO: Dissect this TypeScript magic on the `pro-d` channel on slack
```

Comment:
This would be great to do down the line if we remember.

Reply:
Ok. I'll leave this as unresolved and we can search for it on this PR down the road when it's time. Or better yet, I can commit to dissecting the next similar inferred generic => mapped type I write in the flow of getting stuff done.